src/dotnet-install.ps1

```diff
@@ -658,17 +658,19 @@ function Get-Product-Version([string]$AzureFeed, [string]$SpecificVersion, [stri
         try {
             $productVersionResponse = GetHTTPResponse($productVersionTxtUrl)
 
-            if ($productVersionResponse.StatusCode -eq 200) {
-                $productVersion = $productVersionResponse.Content.ReadAsStringAsync().Result.Trim()
-                if ($productVersion -ne $SpecificVersion)
-                {
-                    Say "Using alternate version $productVersion found in $ProductVersionTxtURL"
-                }
-                return $productVersion
+            if ($productVersionResponse.StatusCode -ne 200) {
+                Say-Verbose "Got unsuccessful StatusCode '$($productVersionResponse.StatusCode)' when trying to get productVersion.txt at $productVersionTxtUrl."
+                continue;
```

Comment:
The semicolon should be removed, same for the other `continue` a few lines below.